webrtc/track.go

```diff
@@ -21,13 +21,32 @@ type RemoteTrack interface {
 	ReadRTP() (*rtp.Packet, error)
 }
 
+// RTPParameters represents RTCRtpParameters which contains information about
+// how the RTC data is to be encoded/decoded.
+//
+// ref: https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpSendParameters
+type RTPParameters struct {
+	SSRC        uint32
```

Comment:
I think we should not include SSRC. `webrtc` package should just randomize this automatically to be less confusing.

Reply:
I think SSRC is also negotiated in SDP as a media attribute. It should be randomly selected before exchanging SDP and passed to encoder.

Reply:
Example of SDP
```
m=video 41065 UDP/TLS/RTP/SAVPF 120 121
c=IN IP4 ****
a=candidate:0 1 *******
...
a=recvonly
a=end-of-candidates
a=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid
...
a=fmtp:120 max-fs=12288;max-fr=60
a=fmtp:121 max-fs=12288;max-fr=60
a=ice-pwd:e6ca50bc3820d4ac138e4d80c5e0403a
a=ice-ufrag:415b3fbb
a=mid:1
a=rtcp:34898 IN IP4 ****
a=rtcp-fb:120 nack
...
a=rtcp-mux
a=rtpmap:120 VP8/90000
a=rtpmap:121 VP9/90000
a=setup:actpass
a=ssrc:3338981088 cname:{87255f94-6b15-4d9d-b43c-2f2748f406e3}
```

I'm not sure which of PeerConnection and RTPSender determines SSRC, but anyway it must be shared between them.

Reply:
I think a ssrc-set-api is useful. the sfu subscriber can use the publisher's ssrc. It is a good way to trace track problem.

Reply:
> I think a ssrc-set-api is useful. the sfu subscriber can use the publisher's ssrc. It is a good way to trace track problem.

Maybe we can use CSRC to include publisher's SSRC?

Reply:
@adwpc in my understanding, SRTP requires SSRC to be random for security reason.
If the one of the sfu publisher make duplication of SSRC, subscriber will cause reuse of encryption IV. Especially with AES-GCM cipher, one corruption of IV cause leak of authentication key.